Resources/Prototypes/_Goobstation/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```
@@ -66,3 +66,47 @@
     radius: 4
     color: purple
     energy: 2
+
+- type: entity
+  categories: [ HideSpawnMenu ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)